### PR TITLE
Added Browser Support to top navigation menu

### PR DESCRIPTION
**Pull request by ijlee2**
## Background

In #745, we created a route called `browser-support`. To help people find this page, we want to add a link to the page to the top navigation menu, between `Releases - Editions` and `Releases - Deprecations`.

The problem is, the data that define the menu links come from the addon [ember-styleguide](https://github.com/ember-learn/ember-styleguide/blob/v5.1.0/addon/constants/links.js). As a result, we need to make a pull request in that repo, then upgrade the addon in `ember-website`.

A complication would arise because we haven't yet upgraded the addon to the latest version. There were breaking changes between `v4` and `v5`.

## Description

In this pull request, I duplicated the data from `ember-styleguide`, then updated the data so that we will show `Browser Support` between `Editions` and `Deprecations`. The update mirrors that in https://github.com/ember-learn/ember-styleguide/pull/359/files.

I think this is a good temporary fix until we can invest time to upgrading `ember-styleguide` to `v5.x`.

### CODE REVIEW

File: app/utils/header-links.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        type: 'link',
+      },
+    ],
+  },
+  {
+    name: 'About',
+    type: 'dropdown',
+    items: [
+      {
+        href: 'https://emberjs.com/teams',
```
Comment (ijlee2) [inline app/utils/header-links.js:132]: Note, in `ember-styleguide`, the value for `href` is `'https://emberjs.com/team'`. We'll want to make another pull request in `ember-styleguide` to fix the URL.

**Review by jaredgalanis — COMMENTED** [review]
Looking at the release history for the styleguide I'm reminded that we made a further release to 4.x of the styleguide to handle dropping jobs.emberjs.com from the community menu. We could do a similar thing here. And release 
Screen_Shot_2021-01-26_at_4_18_50_PM
5.x and 4.x versions of the styleguide.

**Comment by ijlee2** [comment]
@jaredgalanis Yep, I agree that making a minor release in both `v4` and `v5` is ideal.

`deprecation-app` repo currently uses `v4` of the addon, and possibly a couple of other projects. @locks added an agenda item to the Learning Team meeting to discuss further, so let's use that time to plan out what we can do.

**Comment by ijlee2** [comment]
I misunderstood what https://github.com/ember-learn/ember-styleguide/pull/359 was for. I realized just now that it targets the `v4.x` branch so that we can make a minor release in `v4`. I opened a similar issue, https://github.com/ember-learn/ember-styleguide/pull/360.

I think it's better that we close this pull request, review the 2 issues in `ember-styleguide`, and make a minor release for `v4` and `v5`.

**Comment by ijlee2** [comment]
Might have closed the PR too early.
